src/main/appOne.java

```diff
@@ -0,0 +1,11 @@
+class Test{
```

Comment:
line no: 2 - Consider using .equals() method to compare strings instead of == operator for content comparison.